source/reporting/index.html.md.erb

```diff
@@ -149,7 +149,9 @@ Where:
 The response will not contain a `fee` or `net_amount` parameter if either:
 
 - your PSP is SmartPay, Worldpay or ePDQ - we do not deduct these PSPs' transaction fees
-- you're [using a test API key](/quick_start_guide/#test-the-api) - we do not deduct fees from test payments
+- you're using an API key from your test ('sandbox') account - we do not deduct fees from payments in this account
+
+To test transaction fees, you can request a test Stripe account by selecting your account in the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/my-services).
```

Comment:
This is just about avoiding step-by-step button presses - because that would mean the user feeling they have to use the docs as a side-by-side 'guide' for the admin tool (which isn't necessary and would using the admin tool more complicated than it needs to be). It's easier for the user if we 'hand off' to the admin tool UI when it becomes intuitive.

Reply:
It also reduces the risk that the docs become out of date when the UI changes.

Reply:
that makes sense about step by step. Just sounds like we say 'select your account' and somehow that will request it? maybe it's just, you can request a Stripe test account from your account in the GOV.UK Pay admin tool?

Reply:
Ah yes that sounds better - will update